src/java/org/apache/cassandra/io/sstable/format/big/BigFormat.java

```diff
@@ -69,11 +73,19 @@ public class BigFormat extends AbstractSSTableFormat<BigTableReader, BigTableWri
 {
     private final static Logger logger = LoggerFactory.getLogger(BigFormat.class);
 
+    // Feature flag env variable to use extended TTL up to 2106 with the new oa sstable format. See c14227
+    public static enum TTL_MODE {LEGACY_TTL_2038, COMPATIBILITY, EXTENDED_TTL_2106};
+    @VisibleForTesting
+    public static final String USE_OA_SSTABLE_FORMAT = "cassandra.use_oa_sstable_format";
```

Comment:
I'm not sure whether the name of the property should be `use_oa_sstable_format`, when its accepted values are `LEGACY_TTL_2038`, `COMPATIBILITY` and `EXTENDED_TTL_2106`. 

It seems that using `oa` or not is a consequence of the chosen TTL mode. Perhaps the property should be named just `ttl_mode`? So the sstable format will depend on the chosen ttl mode.

Reply:
I've had these same exact thoughts. But a silent sstable format change doesn't sound pretty to me either. We need to come up with a name/values that link the TTL and sstable concepts imo

Reply:
What about `cassandra.sstable.format.ttl_mode`? We already have that prefix on `cassandra.sstable.format.default`, in `CassandraRelevantProperties.SSTABLE_FORMAT_DEFAULT`, added by [CEP-17](https://issues.apache.org/jira/browse/CASSANDRA-17056).

Reply:
That I like a lot.